xpk.py

```diff
@@ -3721,7 +3765,11 @@ def workload_create(args) -> int:
   if setup_docker_image_code != 0:
     xpk_exit(setup_docker_image_code)
 
-  add_env_config(args)
+  tensorboard_config = {}
+  if _VERTEX_TENSORBOARD_FEATURE_FLAG and args.use_vertex_tensorboard:
+    tensorboard_config = create_vertex_experiment(args)
```

Comment:
Lets expose an error code from create_vertex_experiment(args). This way it aligns with the rest of the code as in we only call xpk_exit(1) from the "Main" function like workload_create / cluster_create.

I have a fix in some commit to make add_env_config fit this pattern too.